App.js

```diff
@@ -19,3 +9,20 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
   },
 });
+
+export default class App extends React.Component {
+  constructor() {
```

Comment:
Be wary of using constructors when you don't need them. I would start all components as functional ones at this point.